test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -318,6 +318,46 @@ Describe "TabCompletion" -Tags CI {
             $res.CompletionMatches | Should -HaveCount 1
             $res.CompletionMatches[0].CompletionText | Should -BeExactly $afterTab
         }
+
+        It "Test case insensitive file path" -Skip:(!$IsLinux) {
+            $testFiles = "foo", "Foo", "fOO"
+            $testFiles | ForEach-Object {
+                $filePath = Join-Path $testdrive $_
+                New-Item -ItemType File -Path $filePath
+            }
+            Push-Location $testdrive
+            $beforeTab = "Get-Content f"
+            $res = TabExpansion2 -inputScript $beforeTab -cursorColumn $beforeTab.Length
+            $res.CompletionMatches | Should -HaveCount 3
+
+            # order isn't guaranteed so we'll sort them first
+            $completions = $res.CompletionMatches | Sort-Object CompletionText -CaseSensitive
+            $expected = $testFiles | Sort-Object -CaseSensitive
+
+            for ($i = 0; $i -lt $expected.Count; $i++) {
```

Comment:
We could join with ";" and compare strings. We already used such pattern.

Reply:
Yes, great suggestion.  Makes it easier to see what failed should the test fail rather than failing on the first one.